apps/meteor/app/api/server/v1/teams.ts

```diff
@@ -579,6 +579,14 @@ API.v1.addRoute(
 				return API.v1.failure('Team not found');
 			}
 
+			const canViewInfo =
+				hasPermission(this.userId, 'view-all-teams', teamInfo.roomId) ||
```

Comment:
If this is the same endpoint that normal users use to get the info about a team, this will break it for them.

Reply:
check updated to see if he is an admin or a team member